### DISCUSSION TITLE
Should we release 0.71.7?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#61] [Patches]
Current Release: [0.71.6](https://github.com/facebook/react-native/releases/tag/v0.71.6)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. 
#### Local commits to backport to main

1.

**Comment by kelset** [comment]
once we tested them in 0.72's next RC (RC1) - let's cherry pick @NickGerleman's fixes for TextInput (EditText) (in order):
* https://github.com/facebook/react-native/commit/1743dd7ab40998c4d3491e3b2c56c531daf5dc47
* https://github.com/facebook/react-native/commit/b9e2627d1ce64e8e293e90f743bf13a515db3b4d
* https://github.com/facebook/react-native/commit/cc0ba57ea42d876155b2fd7d9ee78604ff8aa57a
* https://github.com/facebook/react-native/commit/8c9c8ba5adb59f7f891a5307a0bce7200dd3ac7d
* https://github.com/facebook/react-native/commit/0869ea29db6a4ca20b9043d592a2233ae1a0e7a2
* https://github.com/facebook/react-native/commit/5791cf1f7b43aed1d98cad7bcc272d97ab659111
* https://github.com/facebook/react-native/commit/104cb7f81ecbb81841070078c55ddf5d8ea9d24b
* https://github.com/facebook/react-native/commit/b384bb613bf533aebf3271ba335c61946fcd3303
* https://github.com/facebook/react-native/commit/92b898149956a301a44f99019f5c7500335c5553

And before that, this commit: https://github.com/facebook/react-native/commit/8a0fe30591e21b90a3481c1ef3eeadd4b592f3ed

**Reply by efstathiosntonas** [reply]
@kelset can we patch the text input fixes with patch-package or it needs to be built from source? Thanks

**Reply by kelset** [reply]
@efstathiosntonas I'm not sure, and Nick is off on holidays. From taking a look at the files it might be that you can try patching locally. @cortinico what do you think?

**Reply by beetlebum** [reply]
I believe it's not possible to patch package the android code in RN

**Reply by cortinico** [reply]
You'll have to wait for a numbered version or try to build from source with: https://reactnative.dev/contributing/how-to-build-from-source

**Reply by efstathiosntonas** [reply]
Thanks @cortinico

**Reply by kelset** [reply]
PR: https://github.com/facebook/react-native/pull/36874

**Comment by kalyantm** [comment]
@kelset hope you had a great weekend! just curious, when will the samsung text input fixes be cherry picked and patched in a stable version?

**Comment by kelset** [comment]
https://github.com/facebook/react-native/commit/b07cf33279844d30920d8d46edb48b10ed5a1614 to address performance regressions

**Reply by efstathiosntonas** [reply]
@kelset did this ^^ made it to 0.71.7?

**Reply by kelset** [reply]
@efstathiosntonas yes it did 👍

**Comment by renchap** [comment]
https://github.com/facebook/react-native/commit/a804c0f22b4b11b3d9632dc59a6da14f6c4325e3 to fix `onChangeText` firing on `TextInput` mount (fixing regression https://github.com/facebook/react-native/issues/36494)

**Reply by kelset** [reply]
sadly this won't make it in time for 71.7 - the release is in progress as I'm typing - but let's make sure to pick it for the next patch
